ministryofjustice/laa-apply-for-criminal-legal-aid
# Add client has savings page

## Description of change
Add client has savings page

## Link to relevant ticket
https://dsdmoj.atlassian.net/browse/CRIMAPP-211

## Screenshots of changes (if applicable)

### After changes:
<img width="871" alt="image" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/45827968/efefcf08-5126-4436-8866-5cadfa402c79">

## How to manually test the feature

1. rails db:migrate
2. Start an application
3. Go to steps/income/does_client_own_home_land_property
4. Check that selecting No routes to does_client_have_savings_investments
5. Check that submitting with no option selected raises an error
